neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -79,15 +82,17 @@ def __init__(self, api_token=None, proxies=None):
         backend_client = self._get_swagger_client('{}/api/backend/swagger.json'.format(config_api_url))
         self._client_config = self._create_client_config(self.credentials.api_token, backend_client)
```

Comment:
I'm thinking of refactoring this code to be the first call (before getting backend swagger.json) with a static url.

This might allow us to move swagger file at some point, but will still force us to support config endpoint with static path

Reply:
I just changed url of this endpoint in backend and swagger path was never changed, so... I'm not so sure about this. Let's leave it as it is.